library/src/main/java/com/devbrackets/android/exomedia/core/EMListenerMux.java

```diff
@@ -111,7 +118,15 @@ public void onError(EMExoPlayer emExoPlayer, Exception e) {
     }
 
     @Override
-    public void onStateChanged(boolean playWhenReady, int playbackState) {
+    public void onStateChanged(boolean playWhenReady, @PlaybackState int playbackState) {
```

Comment:
In particular there is [NativeVideoView](https://github.com/brianwernick/ExoMedia/blob/master/library/src/main/java/com/devbrackets/android/exomedia/core/video/NativeVideoView.java) for the MediaPlayer backed version, and [NativeMediaPlayer](https://github.com/brianwernick/ExoMedia/blob/master/library/src/main/java/com/devbrackets/android/exomedia/core/audio/NativeMediaPlayer.java) for the `EMAudioPlayer`.  You can probably just listen to the `MediaPlayer.On*Listener`s in the `EMListenerMux` and post the corresponding state.  (I would prefer to actually listen to MediaPlayer states if they exist)